__tests__/app.test.js

```diff
@@ -38,3 +38,38 @@ describe("Get API", () => {
       });
   });
 });
+
+describe("Get API", () => {
+  test("API request should return the object from endpoints.json", () => {
+    return request(app)
+      .get("/api/")
+      .expect(200)
+      .then((result) => {
+        expect(endPoint).toEqual(result.body);
+      });
+  });
+});
+
+describe("Get article by ID", () => {
+  test("A request for a particular id should return an article object with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((result) => {
+        expect(result.body).toHaveProperty("article_id", 1);
+        expect(result.body).toHaveProperty(
+          "title",
+          "Living in the shadow of a great man"
+        );
+      });
+  });
```

Comment:
Good test but we'd like you to check for all of the article properties 🙂 (author, title, article_id, body, article_img_url, topic, created_at, votes).

Reply:
this is a good test but just to echo harrison here, there that this test can cover all the article properties

Reply:
specifically topic, created_at, votes!